sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt

```diff
@@ -217,7 +217,7 @@ class SampleApplication : Application() {
 
     private fun initializeSessionReplay() {
         val shouldUseFgm = SecureRandom().nextInt(100) < USE_FGM_PCT
-        val systemRequirementsConfiguration = SystemRequirementsConfiguration.BASIC
+        val systemRequirementsConfiguration = SystemRequirementsConfiguration.NONE
```

Comment:
why we are doing this change? shouldn't we keep basic to have a case where files are actually accessed in the sample app?

Reply:
We are usually using sample app to make some test on emulator, and emulator by default uses 1 cpu cores, which will not bypass this `BASIC` check.

Reply:
Then maybe use `setMinCPUCores` API instead of pre-built templates?

Reply:
Good idea, updated